code/modules/mapfluff/ruins/objects_and_mobs/cursed_slot_machine.dm

```diff
@@ -0,0 +1,121 @@
+/// Greed's slot machine: Used in the Greed ruin. Deals damage on each use, with a successful use giving a d20 of fate.
+/obj/structure/cursed_slot_machine
+	name = "greed's slot machine"
+	desc = "High stakes, high rewards."
+	icon = 'icons/obj/machines/computer.dmi'
+	icon_state = "slots"
+	anchored = TRUE
+	density = TRUE
+	/// Variable that tracks the screen we display.
+	var/icon_screen = "slots_screen"
+	/// Should we be emitting light?
+	var/brightness_on = TRUE
+	/// The probability the player has to win.
+	var/win_prob = 5
+	/// The maximum amount of curses we will allow a player to have before disallowing them to use the machine.
+	var/max_curse_amount = 5
+	/// clone damaged dealt each roll
+	var/damage_on_roll = 20
```

Comment:
Unused var? This doesn't do clone damage anymore right?

Reply:
Good point